src/NServiceBus.Core/DelayedDelivery/DelayedDelivery.cs

```diff
@@ -2,47 +2,68 @@
 {
     using System;
     using NServiceBus.CircuitBreakers;
+    using NServiceBus.Config;
     using NServiceBus.DelayedDelivery;
     using NServiceBus.DelayedDelivery.TimeoutManager;
     using NServiceBus.DeliveryConstraints;
-    using NServiceBus.Features.DelayedDelivery;
-    using NServiceBus.Settings;
+    using NServiceBus.Pipeline;
     using NServiceBus.Timeout.Core;
+    using NServiceBus.TransportDispatch;
     using NServiceBus.Transports;
 
     /// <summary>
-    /// Used to configure the timeout manager that provides message deferral.
+    /// Allows to delay sent messages.
     /// </summary>
-    public class TimeoutManager : Feature
+    public class DelayedDelivery : Feature
```

Comment:
Is it a general design approach to make features internal and provide the enable/disable `BusConfiguration` extensions?